### ISSUE TITLE
Document how to download only a required slice of refdata for running some unit tests of interest

### ISSUE THREAD

**Issue by jaladh-singhal** [tardis-sn/tardis#2129]
Many of the unit test functions that doesn't have their specific test data stored in the tardis-refdata, still require access to it because of atom data (and other important files?).

Locally cloning the whole [tardis-sn/tardis-refdata](https://github.com/tardis-sn/tardis-refdata) repository is a bit challenging because:
- it requires git lfs to be installed on user's machine otherwise they get very small size files (or their footprints?) that can't be accessed when running the tests.
-  it takes too much time to download >7GB of data.

We should document it in developer docs:
- how to identify what file(s) is being used from refdata (in the problems we were facing with development of unit test in #2119, we identified we needed only [tardis-refdata/atom-data/](https://github.com/tardis-sn/tardis-refdata/blob/master/atom_data/kurucz_cd23_chianti_H_He.h5))
- download only those files, but store then in the directory tree exactly as in tardis-refdata repository
- then run the tests with your refdata slice (this ofc means you're not doing `pytest tardis` but `pytest tardis/path/to/test_file_or_directory_you_are interested_in`)

**Comment by andrewfullard** [comment]
I don't think we need to support this. There is already partial git clone capability in some workflows that users can check.
